app/js/displayPageButtons.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Takes the Array that has been split up and displays the pagination Buttons on the page.
+ * @param paginatedArray
+ * @returns {Promise<void>}
+ */
+async function displayPageBtns(paginatedArray) {
+    let pages = Array.from(paginatedArray.keys()).map(page => ({pageNumber: page + 1}));
+    let buttonsTemplate = await getTemplateAjax('js/templates/paginationButtons.hbs');
+    let template = Handlebars.compile(buttonsTemplate);
+    document.querySelector('.pageSelectors').innerHTML = template({pages});
+}
+
+function pageSelectorFunctionality(HBTemplate, paginatedArrays) {
+    const pages = paginatedArrays.length;
+    let page = 1;
+
+    pageButtonCheck(page, pages);
+
+    document.querySelectorAll('.pageBtn').forEach((btn) => {
+        btn.addEventListener('click', ((evt) => {
```

Comment:
`evt` is never used, so there's no point it being defined.

Reply:
i agree again!